R/build_history_create_new_hist.R

```diff
@@ -27,4 +27,6 @@ build_history_create_new_hist <- function(df_hist, df_new, kpi_number) {
 
   new_hist_db <- new_hist_db |>
     dplyr::arrange(kpi, fin_year, hbres)
+
+  return(new_hist_db)
```

Comment:
I read somewhere that for R return is not required unless it's any early stop, i.e. in an if else statement, you can add the return() in the if and the statement will stop and return the result is the expression is true.
This is obviously not an issue here, but depending what comes after this part of the function you could just remove the assign statement (new_hist_db <-) and the new_hist_db will automatically be returned after the arrange statement.

new_hist_db |>
    dplyr::arrange(kpi, fin_year, hbres)

Reply:
cool! that makes sense, have changed this